patch-package
--- a/node_modules/firebase-admin/lib/app/credential-internal.js
+++ b/node_modules/firebase-admin/lib/app/credential-internal.js
@@ -17,6 +17,10 @@
  * @license
  * Copyright 2017 Google Inc.
  *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
  *   http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
@@ -24,6 +28,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const {setFipsAware} = require('crypto');
+setFipsAware(false);
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.ServiceAccountCredential = exports.RefreshTokenCredential = exports.ApplicationDefaultCredential = exports.GoogleOAuthCredential = void 0;
 const deep_copy_1 = require("../utils/deep-copy");
